packages/documentation-v7/src/stories/components/button-group/button-group.stories.ts

```diff
@@ -0,0 +1,245 @@
+      options: [1, 2, 3],
+      table: {
+        category: 'Value',
+      },
+    },
+    selected: {
+      name: 'Selected Button',
+      description: 'Defined which button is selected in a checkbox group.',
+      if: {
+        arg: 'element',
+        eq: 'checkbox',
+      },
+      control: {
+        type: 'inline-check',
+        labels: {
+          1: 'First',
+          2: 'Second',
+          3: 'Third',
+        },
+      },
+      options: [1, 2, 3],
+      table: {
+        category: 'Value',
+      },
+    },
+  },
+  decorators: [
+    story =>
+      html`
+        <div
+          @click=${(e: Event) => {
+            const target = e.target as HTMLElement;
+            if (target.tagName === 'A' || target.tagName === 'BUTTON') e.preventDefault();
+          }}
+        >
+          ${story()}
+        </div>
+      `,
+  ],
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+function createButttonTemplate(args: Args, context: StoryContext, index: number) {
+  const [_, updateArgs] = useArgs();
+  const position = index + 1;
+  const id = `btngroup${context.name}_${position}`;
+  const label = args[`label_${position}`];
+  switch (args.element) {
+    case 'checkbox': {
+      const isSelected = args.selected?.includes(position) ?? false;
+      return html`
+        <input
+          type="checkbox"
+          class="btn-check"
+          id=${id}
+          autocomplete="off"
+          ?checked="${isSelected}"
+          .checked="${isSelected}"
+          @change=${(e: Event) => {
+            let isChecked = [];
+            if ((e.target as HTMLInputElement).checked === true) {
+              if (args.selected) {
+                isChecked = Array.from(new Set(args.selected.concat(position)));
+              } else {
+                isChecked = [position];
+              }
+            } else {
+              isChecked = args.selected.filter((p: number) => p !== position);
+            }
+            updateArgs({ selected: isChecked });
+          }}
+        />
+        <label class=${`btn${args.size} btn-secondary`} for=${id}>${label}</label>
+      `;
+    }
+    case 'radio': {
+      const isChecked = position === args.checked;
+      return html`
+        <input
+          type="radio"
+          class="btn-check"
+          name="btnradio"
+          id=${id}
+          autocomplete="off"
+          ?checked=${isChecked}
+          .checked=${isChecked}
+          @change=${(e: Event) => {
+            updateArgs({ checked: position });
+          }}
+        />
+        <label class=${`btn${args.size} btn-secondary`} for=${id}>${label}</label>
+      `;
+    }
+    case 'link':
+      return html`
+        <a href="#" class=${`btn${args.size} btn-secondary`}>${label}</a>
+      `;
+    case 'button':
+    default:
+      return html`
+        <button type="button" class=${`btn${args.size} btn-secondary`}>${label}</button>
+      `;
+  }
+}
+function renderButtonGroup(args: Args, context: StoryContext) {
```

Comment:
```suggestion

function renderButtonGroup(args: Args, context: StoryContext) {
```
For better readiness